Library/Homebrew/tap.rb

```diff
@@ -130,9 +130,7 @@ def git_last_commit_date
   # The issues URL of this {Tap}.
   # e.g. `https://github.com/user/homebrew-repo/issues`
   def issues_url
-    if official? || !custom_remote?
-      "https://github.com/#{user}/homebrew-#{repo}/issues"
-    end
+    "https://github.com/#{user}/homebrew-#{repo}/issues" if official? || !custom_remote?
```

Comment:
Better to try and keep this below 80 chars by e.g. splitting this into an`if .. end`.